diff --git a/utils/kamctl/kamctl b/utils/kamctl/kamctl
index fbb237e..d08397e 100755
--- a/utils/kamctl/kamctl
+++ b/utils/kamctl/kamctl
@@ -277,7 +277,7 @@ is_user() {
 	QUERY="select count(*) from $SUB_TABLE where \
 $SUBSCRIBER_COLUMN='$OSERUSER' and $REALM_COLUMN='$OSERDOMAIN';"
 
-	CNT=`$DBROCMD "$QUERY" "$DBRAWPARAMS" | $EGREP -v ERROR | $LAST_LINE`
+	CNT=`$DBROCMD "$QUERY" "$DBRAWPARAMS" | $EGREP -v ERROR | $LAST_LINE -1`
 	mdbg "is_user: user counter=$CNT"
 	if [ "$CNT" = "0" ] ; then
 		false
@@ -295,7 +295,7 @@ is_value_in_db() {
 	VALUE=$3
 
 	QUERY="select count(*) from $TABLE where $COL='$VALUE';"
-	CNT=`$DBROCMD "$QUERY" "$DBRAWPARAMS" | $EGREP -v ERROR | $LAST_LINE`
+	CNT=`$DBROCMD "$QUERY" "$DBRAWPARAMS" | $EGREP -v ERROR | $LAST_LINE -1`
 	mdbg "is_value_in_db: counter=$CNT"
 	if [ "$CNT" = "0" ] ; then
 		false
@@ -458,7 +458,7 @@ check_db_alias() {
 
 	QUERY="select count(*) from $DA_TABLE where $DA_ALIAS_USER_COLUMN='$1' \
 and $DA_ALIAS_DOMAIN_COLUMN='$2';"
-	CNT=`$DBROCMD "$QUERY" | $EGREP -v ERROR | $LAST_LINE`
+	CNT=`$DBROCMD "$QUERY" | $EGREP -v ERROR | $LAST_LINE -1`
 	mdbg "check_db_alias: alias counter=$CNT"
 	if [ "$CNT" = "0" ] ; then
 		ALIAS_DB_EXISTS=0
